client/www/pages/docs/strong-init.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Strong Init (experimental)
+---
+
+We're piloting strictly-typed variants of our client libraries that leverage your `instant.schema.ts` definition. We're calling it "Strong Init". Strong Init enhances InstaQL and InstaML methods with deep typesafety and enhances your editor experience with rich intellisence.
```

Comment:
```suggestion
What if you could use the types you defined in `instant.schema.ts`, inside `init`? We have a new version of `init` out that lets you do this. It's in beta, but if you use it, `useQuery` and `transact` will automatically get typesafety and intellisense.

Here's how it works
```

Main thoughts: 
- I would try to use simpler words that you would use in conversation with a developer. i.e 'leverage', 'variant', 'client libraries', 'enhances', 'rich', feel a bit out of place
- I would avoid the word 'simply', as it could get the developer who is confused thinking 'if it's so simple, why am I confused'